rules-reviewed/technology-usage/tests/data/non-xml/batch.xml

```diff
@@ -0,0 +1,3 @@
+<batch-artifacts xmlns="http://xmlns.jcp.org/xml/ns/javaee">
```

Comment:
As you modified the rule to consider this kind of xml files and use {*} , should the test follow that approach and then rename this file to : whatever.xml ? In order to see that the modification is working.

Reply:
I've changed it back to batch.xml -  this was altered when I was struggling to get the rule to fire, but I forgot to replace the filename once I got it working.